README.md

```diff
@@ -1,2 +1,5 @@
+![](./workflows?branch=master/Build/badge.svg)
+![](./workflows/Deploy/badge.svg)
```

Comment:
Wouldn't we want this to be `branch=master` too?

Reply:
It should default to whatever was last deployed. The other one runs for PRs too, so I want to specify status for the last build on `master`

Reply:
Makes more sense after reading the other comment 👍